frontend/src/app/selectors/experiment.js

```diff
@@ -0,0 +1,30 @@
+import { createSelector } from 'reselect';
```

Comment:
Also, Dan's advice you link is still good here: it's about decoupling from actions and reducers, not from keeping them in separate files.